### PR TITLE
Fix issue

### PR DESCRIPTION
**Pull request by DavidJayakumar**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/EUI-8923

### Change description ###

Fix issue with display of other option

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [x] Does this PR introduce a breaking change

### CODE REVIEW

File: src/register-org/components/organisation-type/organisation-type.component.ts [hunk 1]
```diff
@@ -47,6 +47,8 @@ export class OrganisationTypeComponent extends RegisterComponent implements OnIn
     }
 
     this.subscription = this.lovRefDataService.getListOfValues(this.CATEGORY_ORGANISATION_TYPE, true).subscribe((orgTypes) => {
+      // To set Other option as the last option
+      orgTypes.push(orgTypes.splice(orgTypes.findIndex((e) => e.key === 'OTHER'), 1)[0]);
```
Comment (udaydenduluri33) [inline src/register-org/components/organisation-type/organisation-type.component.ts:51]: Can you please rewrite the logic.

1. Create a new function that takes old Array and returns an old Array
2. Copy all elements from old array to new array except OTHER
3. If there is other then have the last element as OTHER

**Review by udaydenduluri33 — APPROVED** [review]

**Review by udaydenduluri33 — CHANGES_REQUESTED** [review]
Comments below

File: src/register-org/components/organisation-type/organisation-type.component.ts (hunk 1 above)
Comment (DavidJayakumar) [reply src/register-org/components/organisation-type/organisation-type.component.ts]: Moved the logic to app util

**Review by udaydenduluri33 — APPROVED** [review]
LGTM
Thanks for making the changes

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Pull request merged**